### PR TITLE
chore(readme): Make README.md friendlier without giving out many details

### PR DESCRIPTION
**Pull request by tklarryonline**
Improved the README.md files by:

- Fixing typos and removing dead links
- Rewriting the messages without giving out too many details
- Increasing the visibility of the functional requirements

### CODE REVIEW

File: fullstack/README.md [hunk 1]
```diff
@@ -1,18 +1,29 @@
...
+  or Ruby, and use their respective folder in the pairing session.
+- Set up the app in your IDE of choice. You will be sharing your screen during
+  the pairing interview.
+- Make sure you can build and run the code.
+- Familiarise yourself with
+  - Code structure
+  - Tests
+  - Business logic
+  - Design choices
+- It's not perfect, we'd love to hear how you've improve it
```
Comment (bjwrk) [inline fullstack/README.md]:
```suggestion
- It's not perfect, we'd love to hear how you'd improve it
```

**Review by bjwrk — APPROVED** [review]
LGTM, apart from one tiny typo

**Comment by cosmini** [comment]
**Really love this initiative!** ❤️ 

In my opinion we should observe consistency between the read me files in the fullstack folder:
- [go price-calculator](https://github.com/redbubble/pairing-exercises/tree/fix-readme-typos/fullstack/go/price-calculator)
- [python price_calculator](https://github.com/redbubble/pairing-exercises/tree/fix-readme-typos/fullstack/python/price_calculator)
- [ruby pricecalculator](https://github.com/redbubble/pairing-exercises/tree/fix-readme-typos/fullstack/ruby/pricecalculator)

For example the title, sections and available commands are fairly different which may confuse someone.
Plus each dir name is inconsistent: kebab case vs snake case vs nocase 🙈

What do you think if we simplify this?

Here are some sections I'd say we need:
- title
- functional requirements
- requirements
- installation
- usage
- test

**Comment by tklarryonline** [comment]
Let's just make sure the sections are in sync among the language options. We can rename the dirs in another PR.

**Review by cosmini — APPROVED** [review]

File: fullstack/README.md [hunk 2]
```diff
@@ -1,18 +1,29 @@
...
 ## Context
 
-The code under the `price-calculator` folder will be used as a basis for our fullstack pairing exercise.
-In the interview we'll talk through an extension and your thoughts on design and how you might evolve it.
+Each language has its own implementation of a Price Calculator app based on the
+following [functional requirements](http://take-home-test.herokuapp.com/new-product-engineer).
+We'll use it as a basis for our fullstack pairing exercise.
+
+In the interview we'll talk through the structure of the codebase, your thoughts
+on the design, and how you might involve it. Then we will do an extension on the
```
Comment (chris-lumsden-rb) [inline fullstack/README.md]: Did you mean this?
```suggestion
on the design, and how you might improve it. Then we will do an extension on the
```

File: fullstack/go/price-calculator/README.md [hunk 3]
```diff
@@ -1,16 +1,26 @@
...
-* coverage `./build.sh coverage`
-
-###Alternatively
-* to build from the project root `GOBIN=$(pwd)/bin go install ./...` 
-* then run `./bin/calculate -cartFile=<cartFile> -pricesFile=<priceFile>`
+# Redbubble Fullstack Pairing Exercise - Price Calculator
+
+## Functional Requirements
+
+You can find the function requirements at the following URL:
```
Comment (chris-lumsden-rb) [inline fullstack/go/price-calculator/README.md]:
```suggestion
You can find the functional requirements at the following URL:
```

File: fullstack/ruby/pricecalculator/README.md [hunk 4]
```diff
@@ -1,76 +1,93 @@
-# Redbubble Coding Test - Price Calculator
+# Redbubble Fullstack Pairing Exercise - Price Calculator
 
-### Development Notes
-[Some thoughts before, during and after development](./NOTES.md)
+## Functional Requirements
+
+You can find the function requirements at the following URL:
```
Comment (chris-lumsden-rb) [inline fullstack/ruby/pricecalculator/README.md]:
```suggestion
You can find the functional requirements at the following URL:
```

File: fullstack/python/price_calculator/README.md [hunk 5]
```diff
@@ -1,24 +1,26 @@
 # Skeleton for Redbubble's Pairing Exercise
 
+## Functional Requirements
+
+You can find the function requirements at the following URL:
```
Comment (chris-lumsden-rb) [inline fullstack/python/price_calculator/README.md]:
```suggestion
You can find the functional requirements at the following URL:
```

**Review by chris-lumsden-rb — APPROVED** [review]
Great stuff 😍 A slick candidate experience is so important, thank you for this 🙇‍♂️

File: fullstack/README.md (hunk 2 above)
Comment (tklarryonline) [reply fullstack/README.md]: Good catch! I was thinking about `evolve` but my fingers were involved in the typing 😅

**Pull request merged**